index.bs

```diff
@@ -2778,6 +2778,52 @@ enum ChannelInterpretation {
 </table>
 </div>
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn>. This means that even when the
+{{AudioNode}} is fed silence, non-silent output can be output.
+
+{{AudioNode}}s have a non-zero tail-time if they have internal processing state
+such that input received at one time affects the output at later times. Such
+{{AudioNode}}s that have received non-silent input continue to produce
+non-silent output for the length of the tail-time after the input becomes
+silent.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
+during a <a>render quantum</a>, if any of the following conditions hold.
+
+- An {{AudioScheduledSourceNode}} is [=actively processing=] if and only if it
+	is playing for at least part of the current rendering quantum.
+- A {{MediaElementAudioSourceNode}} is [=actively processing=] if and only if its
+	{{MediaElementAudioSourceNode/mediaElement}} is playing for at least part of
+	the current rendering quantum.
+- A {{MediaStreamAudioSourceNode}} or {{MediaStreamTrackAudioSourceNode}} when
+	the [[mediacapture-streams#mediastreamtrack|MediaStreamTrack]] used as input
+	has a <code>readyState</code> of <code>live</code>, a <code>muted</code>
+	attribute that is <code>false</code> and an <code>enabled</code> attribute
+	that is <code>true</code>.
+- A {{DelayNode}} in a cycle is [=actively processing=] only when the absolute value
+	of any output sample for the current [=render quantum=] is greater than or equal
+	to \( 2^{-126} \).
+- A {{ScriptProcessorNode}} is [=actively processing=] when its input or output is
+	connected.
+- An {{AudioWorkletNode}} is [=actively processing=] when its
+	{{AudioWorkletProcessor}}'s {{[[callable process]]}} is <code>true</code> and
```

Comment:
You mean "returns true" instead of "is true"?

Reply:
Yes.